### ISSUE TITLE
Warmup causes NAN

### ISSUE THREAD

**Issue by lyw615** [CyberZHG/keras-radam#43]
I use RAdam in maskrcnn  with keras implement. But after warmup completed,  the loss value get NAN. If  SGD used, without warmup, the loss value is normal. I just do the operation during heads layers training. Usage is as following, and any reply will be appreciated:   learning_rate is 0.001.
 ```
if warmup:
    optimizer_use=RAdam(learning_rate=1e-5,total_steps=all_steps, 
  warmup_proportion=0.05,min_lr=learning_rate)`
            
 else:
            
    optimizer_use=RAdam()

 self.keras_model.compile(optimizer=optimizer_use, loss=[ None] * len(self.keras_model.outputs))

```
